packages/maskbook/src/plugins/Trader/apis/coinmarketcap/getPriceStat.ts

```diff
@@ -0,0 +1,125 @@
+import { EventEmitter } from 'events'
+import { CMC_LATEST_BASE_URL } from '../../constants/trending'
+
+const MAX_RETRY = 3
+let webSocket: WebSocket | null = null
+let retry = MAX_RETRY
+const Event = new EventEmitter()
+
+export interface PriceState {
+    id: number
+    mc: number
+    p: number
+    p1h: number
+    p7d: number
+    p24h: number
+    v: number
```

Comment:
would you like to add `/** */` style comment to notify people what thoes fields are